docs/source/getting-started/tabular-data/index.rst

```diff
@@ -52,52 +59,103 @@ The `User Guide <https://ec.europa.eu/eurostat/documents/1978984/6037342/EULFS-D
-       split_evenly_over=10,
-       margins={
-           ("YEAR", ): dp.polars.Margin(max_partition_length=60_000_000, max_partition_contributions=4),
-           ("YEAR", "QUARTER",): dp.polars.Margin(max_partition_length=60_000_000, max_partition_contributions=1),
-           (): dp.polars.Margin(max_partition_length=60_000_000),
-       },
-   )
-
-**Parameters**:
 
-* *privacy_unit:* How many rows each individual or entity of interest contributes to our data frame. In this case, we are analyzing the data from across 13 years and each year has 4 quarters. Therefore, the unit of privacy is 36. If we were to analyze a particular quarter in a particular year, the unit of privacy would be 1 since each individual would be represented once. 
-
-* *privacy_loss:* This parameter determines how much privacy we want to preserve. If ε is small, we will have more privacy but worse data accuracy. ε can range from 0 to infinity, but 1 is usually a standard. 
+Overview
+----------
 
-* *split_evenly_over:* This is the number of queries you want to distribute your privacy loss over. For now we specified 10 to explore the API but in the final versions of your code, this parameter will be picked more carefully. 
+The specific methods that will be demonstrated are: 
 
-* *margins:* Margins capture the variables of interest in your analysis. This can include variables that you may want to group by or apply differential privacy techniques to. 
-    * *max_partition_length:* The upper bound on how many records (individuals in this case) can be in one partition. If you do not know the size of your dataset, this can be an upper bound on the population represented in your dataset. The population of France was about 60 million in 2004 so that's our maximum partition length. Source: `World Bank <https://datatopics.worldbank.org/world-development-indicators/>`_. 
-    * *max_partition_contributions:* The number of contributions each individual can have per grouping. Since each individual is represented once for a particular quarter and year, they are represented 13 times for each quarter since there are 13 years in the dataset and 4 times each year since there are 4 quarters within a year. 
+* Fundamental Statistics 
+    * Count
+    * Sum 
+    * Mean 
+    * Median 
+    * Quantiles 
+* Grouping
+    * Grouping By Multiple Variables 
+    * Filtering
+    * Public vs. Private Grouping Keys
+* Data Preparation
+    * Limitations with ``with_columns``
+    * Limitations with ``filter`` 
 
-Particular examples will require additional parameters, and the compositor will change slightly.
+Compositor Overview
+-------------------
+The compositor is the foundation of our differentially private analysis. 
+It mediates access to the sensitive data,
+ensuring that queries you would like to release satisfy necessary privacy properties. 
+
+.. testsetup::
+    >>> import polars as pl
+    >>> df = pl.LazyFrame()
+
+.. doctest:: python
```

Comment:
As long as we're clear that sphinx doesn't have a role in the doc testing, great: If I'm confused, let me know.